sendwp/installer.js

```diff
@@ -3,24 +3,37 @@ function caldera_forms_sendwp_remote_install() {
         'action': 'caldera_forms_sendwp_remote_install',
         'sendwp_nonce': sendwp_vars.nonce
     };
+    
     // since 2.8 ajaxurl is always defined in the admin header and points to admin-ajax.php
     jQuery.post(ajaxurl, data, function(response) {
         var data = JSON.parse(response);
-        //Check for errors before calling caldera_forms_sendwp_register_client()
-        if(data.error === true ){
+         
+         if(data.error === true ){
```

Comment:
@New0 In my local testing. The response is `{error: 'Something went wrong ..."}` when it fails, which explains why I am getting the redirect to `/undefined`. I think this could just be `if(data.errror){...`